jenkins-setup/setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+JENKINS_ROOT=./jenkins
+JENKINS_VERSION=2.319.3
+JENKINS_PM_VERSION=2.5.0
+JFR_VERSION=1.0-beta-30
+
+if [ $# == 1 ]
+then
+    JENKINS_VERSION=$1
+fi
+
+JENKINS_PM_URL="${JENKINS_PM_URL:-https://github.com/jenkinsci/plugin-installation-manager-tool/releases/download/${JENKINS_PM_VERSION}/jenkins-plugin-manager-${JENKINS_PM_VERSION}.jar}"
+JENKINS_CORE_URL="${JENKINS_CORE_URL:-http://updates.jenkins.io/download/war/${JENKINS_VERSION}/jenkins.war}"
+JENKINS_JFR_URL="${JENKINS_JFR_URL:-https://github.com/jenkinsci/jenkinsfile-runner/releases/download/${JFR_VERSION}/jenkinsfile-runner-${JFR_VERSION}.zip}"
+
+# download Jenkins core
+mkdir -p ${JENKINS_ROOT}
+echo "Downloading Jenkins $JENKINS_VERSION core..."
+curl -s -L "${JENKINS_CORE_URL}" -o ${JENKINS_ROOT}/jenkins.war
+# download Jenkinsfile-runner
+echo "Downloading Jenkinsfile runner..."
+curl -s -L "${JENKINS_JFR_URL}" -o ${JENKINS_ROOT}/jenkinsfile-runner-${JFR_VERSION}.zip
```

Comment:
I would suggest using long options whenever possible for better readability. For example, replace `curl -s` with `curl --silent`.

Reply:
Yeah, being explicit is better than using the short form for flags